### ISSUE TITLE
[BUG] Bootstrap produced error lines of ...not found zroot/zroot/bastille...

### ISSUE THREAD

**Issue by jemail99** [BastilleBSD/bastille#754]
**[MANDATORY] Describe the bug [MANDATORY]**
Using the following in bastille.conf produced bootstrap errors of not found zroot/zroot/bastille...
```
## ZFS options
bastille_zfs_enable="YES"                                             ## default: ""
bastille_zfs_zpool="zroot"                                            ## default: ""
bastille_zfs_prefix="${bastille_zfs_zpool}/bastille"   ## default: "${bastille_zfs_zpool}/bastille"
```

Changing options to 
```
bastille_zfs_enable="YES"                                             ## default: ""
bastille_zfs_zpool="zroot"                                            ## default: ""
bastille_zfs_prefix="bastille"                                        ## default: "${bastille_zfs_zpool}/bastille"
```
allowed bootstrap command to work correctly.

Using options
```
bastille_zfs_enable="YES"                                             ## default: ""
bastille_zfs_zpool=""                                                 ## default: ""
bastille_zfs_prefix="${bastille_zfs_zpool}/bastille"                  ## default: "${bastille_zfs_zpool}/bastille"
```
produces zfs options not set error.

**[MANDATORY] Bastille and FreeBSD version (paste ``bastille -v && freebsd-version -kru`` output)**
bastille -v && freebsd-version -kru
0.10.20231125  <- note this is wrong see my other bug report, it seems to actually be 0.11.20241022
14.1-RELEASE
14.1-RELEASE
14.1-RELEASE

**[MANDATORY] How did you install bastille? (port/pkg/git)**
Installed using package

**[optional] Steps to reproduce?**
`bastille bootstrap 14.1-RELEASE update`

**[optional] Expected behavior**
Seems default options are not working correctly?

**[optional] Screenshots**

**[optional]  Additional context**

**Comment by tschettervictor** [comment]
We are aware of this and have fixed it in the 0.12 version.

See #713

**Comment by yaazkal** [comment]
Yes @jemail99 as @tschettervictor said.

Please use the latest version https://github.com/BastilleBSD/bastille/releases/tag/0.12.20241124

Regards.
